Connection/RESTConnector.cs

```diff
@@ -259,7 +259,6 @@ public Request()
         /// This function returns a RESTConnector object for the given service and function.
         /// </summary>
         /// <param name="authenticator">Authenticator used to authenticate service.</param>
-        /// <param name="function">The name of the function.</param>
```

Comment:
Was removing this intentional? It is still in the operation params

Reply:
unintentional, I was initially updating the method and decided not to. Let me update.